ui/perfherder/helpers.js

```diff
@@ -1,11 +1,24 @@
 import chunk from 'lodash/chunk';
+import numeral from 'numeral';
+import padStart from 'lodash/padStart';
```

Comment:
This usage of `lodash/padStart` has been removed on `master` - it looks like the changes were lost in the moving of code to this file?

Reply:
Oh, you're right. Fixed that.